app/src/main/java/org/fossasia/phimpme/gallery/util/PermissionUtils.java

```diff
@@ -25,4 +25,13 @@ public static void requestPermissions(Object o, int permissionId, String... perm
       ActivityCompat.requestPermissions((AppCompatActivity) o, permissions, permissionId);
     }
   }
+
+  public static boolean shouldShowPermissionRationale(
+      Object o, int permissionId, String permission) {
```

Comment:
Why is this receiving object?

Reply:
Has always been receiving object, will change it.